test/models/role_test.rb

```diff
@@ -36,6 +36,15 @@ class RoleTest < ActiveSupport::TestCase
     role.must_be :valid?
   end
 
+  it "Should delete role who has users" do
```

Comment:
Fixed the case on merge, but I'd prefer in this case to keep the objects persisted in the DB to ensure we test the whole destroy mechanism (there are a few dependent destroys with cached user roles etc).